package.json

```diff
@@ -52,7 +52,7 @@
   "dependencies": {
     "jquery": "^3.3.1",
     "leaflet": "^1.3.1",
-    "leaflet-blurred-location": "git://github.com/publiclab/leaflet-blurred-location#main",
+    "leaflet-blurred-location": "git://github.com/publiclab/leaflet-blurred-location.git#main",
     "leaflet-fullhash": "github:sagarpreet-chadha/leaflet-fullHash",
```

Comment:
`leaflet-fullhash` @nstjean @crisner 
We can remove this package now, right?

Reply:
Which one, `leaflet-fullhash`?

Reply:
Yes, It is not used in LEL anymore.